.github/workflows/tests_development.yml

```diff
@@ -11,6 +11,9 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    defaults:
```

Comment:
Could we also try adding a basic platform strategy as described [here](https://github.com/actions/setup-python)? I think for now something like the below will be fine, so we test the two main OSs which we know are used and the current and future major python versions.

```yaml
runs-on: ${{ matrix.os }}
    strategy:
      matrix:
        os: [ubuntu-latest, macos-latest]
        python-version: ['3.9', '3.10']
```